SECURITY.md

```diff
@@ -0,0 +1,74 @@
+# Security Policy
+
+This document describes the Security Policy for ApeWorX Ltd.,
+including vulnerability disclosures for the [Ape Framework](https://github.com/ApeWorX/ape)
+and any of it's [maintained plugins](https://github.com/ApeWorX?q=%22ape-%22&type=public&language=&sort=).
+We are committed to conduct our Security Process in a professional and civil manner.
+Public shaming, under-reporting, or misrepresentation of vulnerabilities will not be tolerated.
+
+To submit a finding, please follow the steps outlined in receiving disclosures [section](#disclosing-a-vulnerability).
+
+## Responsible Disclosure Standard
+
+ApeWorX follows a community [standard](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#the-standard)
+for responsible disclosure in cryptocurrency and related software.
+This document is a public commitment to following the standard.
+
+This standard provides detailed information for:
+
+- [Initial Contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact):
+  how to establish initial contact with our security team.
+- [Giving Details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details):
+  what details to include with your vulnerability disclosure after having received a response to your initial contact.
+- [Setting Dates](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#setting-dates):
+  how to agree on timelines for releasing updates and making details of the issue public.
+
+Any expected deviations and necessary clarifications around the standard are explained in the following sections.
+
+## Disclosing a Vulnerability
+
+### Supported Versions
+
+Ape is currently under active development, targeting a 1.x release.
+Any security issues found on the latest minor version of Ape will be fixed and released in either
+a patch or minor release.
```

Comment:
The formatting of this document is veering from our norm of always doing 1 line per sentence in markdown.